src/applications/disability-benefits/526EZ/tests/components/DisabilityWizard.unit.spec.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import createCommonStore from '../../../../../platform/startup/store';
+import DisabilityWizard from '../../components/DisabilityWizard';
+import { layouts } from '../../helpers';
+
+const { chooseUpdate, applyGuidance } = layouts;
+
+const defaultProps = {
+  store: createCommonStore()
+};
+
+describe('<DisabilityWizard>', () => {
+  it('should show button and no questions', () => {
+    const tree = mount(
+      <DisabilityWizard {...defaultProps}/>
+    );
+
+    expect(tree.find('button')).not.to.be.false;
+    expect(tree.find('#wizardOptions').props().className).to.contain('wizard-content-closed');
+  });
+  it('should show update page', () => {
+    const tree = mount(
+      shallow(
```

Comment:
Why nest `mount(shallow(<Component>))`? If you can't get away with just `shallow` (and I think you maybe can't here), using only `mount` should do the trick.

Reply:
`shallow` is necessary for `setState`, and `mount` is required to use `props`.